create_config.py

```diff
@@ -1,21 +1,24 @@
-from jinja2 import Template
+import os
 import json
+from jinja2 import Template
 
+DEBUG = True
+output_dir = "output_configs"
 
+with open('ConfTemplate.j2', 'r') as fp:
+    loadtemplate = fp.read()
 
-with open('ConfTemplate.txt','r') as fp:
-    loadtemplate=fp.read()
-
-with open('Clients.txt') as json_file:
-    clients=json.load(json_file)
-
-jinjatemplate=Template(loadtemplate)
+with open('Clients.json', 'r') as json_file:
+    clients = json.load(json_file)
 
+jinjatemplate = Template(loadtemplate)
 
-for key in clients:
-    hostname=key
-    ip=clients[key]
-    conf=jinjatemplate.render(hostname=hostname,ip=ip)
-myfile=open(key+'.conf','w') #to move in the for loop
-result=myfile.writelines(conf)#to move in the for loop
+os.makedirs(output_dir, exist_ok=True)
```

Comment:
ok but I prefer as I did. I am not sure about to create a new folder because after that you have run the script you have to unpack all files and delate the folder

Reply:
See my comment for line 6, we can change it if you prefer